storm-core/src/clj/backtype/storm/daemon/nimbus.clj

```diff
@@ -14,15 +14,29 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.daemon.nimbus
-  (:import [org.apache.thrift.server THsHaServer THsHaServer$Args])
+  (:import [org.apache.thrift.server THsHaServer THsHaServer$Args]
+           [backtype.storm.generated KeyNotFoundException]
+           [backtype.storm.blobstore LocalFsBlobStore])
   (:import [org.apache.thrift.protocol TBinaryProtocol TBinaryProtocol$Factory])
   (:import [org.apache.thrift.exception])
   (:import [org.apache.thrift.transport TNonblockingServerTransport TNonblockingServerSocket])
   (:import [org.apache.commons.io FileUtils])
+  (:import [javax.security.auth Subject])
   (:import [java.nio ByteBuffer]
            [java.util Collections HashMap]
            [backtype.storm.generated NimbusSummary])
-  (:import [java.io FileNotFoundException File FileOutputStream])
+  (:import [backtype.storm.security.auth NimbusPrincipal])
+  (:import [java.util Iterator])
+  (:import [java.nio ByteBuffer]
+           [java.util Collections List HashMap])
+  (:import [backtype.storm.blobstore AtomicOutputStream
+            BlobStore
+            BlobStoreAclHandler
+            ClientBlobStore
+            InputStreamWithMeta
+            KeyFilter
+            SyncBlobs])
+            (:import [java.io FileNotFoundException File FileOutputStream FileInputStream])
```

Comment:
The indentation here looks off, and in other places we don't do one Class per line for an import, so it would be nice to mirror that here.